tests/integ/test_tf_script_mode.py

```diff
@@ -61,7 +66,7 @@ def test_mnist_with_checkpoint_config(
         sagemaker_session=sagemaker_session,
         script_mode=True,
         framework_version=tf_full_version,
-        py_version=py_version,
+        py_version="py37",
```

Comment:
what are all of the conditions where we want "py37"? I'm a little confused that the fixture is being modified while at the same time we're switching to not using it in specific tests

Reply:
My bad, I had tested "py37" and modified the ``py_version`` fixture but forgot the change it back..

Reply:
talked offline - we can leave it hardcoded